src/BladeEngineWrapper.php

```diff
@@ -0,0 +1,132 @@
+<?php 
```

Comment:
Cache path is defined multiple times. Would be nice if we make a class var to define location.